tools/aqrdev

```diff
@@ -200,26 +200,52 @@ def cmd_create(
         click.secho(f"Deployment '{name}' already exists", fg="red")
         sys.exit(errno.EEXIST)
 
+    """
+    User did not assign box or image
+    1) search for existing default aquarium box and provider type, then flag it
+    2) look up available image name and provider type, then flag it 
+    3) if neither exist, error out
```

Comment:
nit: 
Still spelling mistakes: "If neither exists, then exit with error".
The overall comment does not help to understand the code, like what does it mean "flag it"?
Probably we can just drop it or rework.

Reply:
@kshtsk my comment is added so I can make sense out of the logic. flag it mean exactly that and we need to change the flag later. Because the logic need to test if the box exist, image exist or neither.

Reply:
when I grep the code with 'flag' keyword I see nothing, if under 'flag it' you mean this `aquarium_box_exist = True` then I don't see how this details helps me as a code reader (not the author) to better understand the general logic of this block.
I usually expect from the comment a better understanding of the code or quick overview so I do not spend my time and just skip it.